src/pages/Admin/Venue/DetailsForm.tsx

```diff
@@ -52,6 +53,11 @@ import { Form } from "react-bootstrap";
 import QuestionInput from "./QuestionInput";
 import EntranceInput from "./EntranceInput";
 import { ImageCollectionInput } from "components/molecules/ImageInput/ImageCollectionInput";
+import { UserStatusManager } from "components/molecules/UserStatusManager";
+import { UserStatus } from "types/User";
+import { useConnectCurrentVenueNG } from "hooks/useConnectCurrentVenueNG";
+import { useSovereignVenueId } from "hooks/useSovereignVenueId";
+import { useShowHide } from "hooks/useShowHide";
```

Comment:
would be cool to sort out the imports if you have the capacity. 🙃

Reply:
Changed in: https://github.com/sparkletown/sparkle/pull/1532/commits/0c1f9f07a5696848014d78ad2604de3c1ed2d194